insights/parsers/ironic_conf.py

```diff
@@ -0,0 +1,56 @@
+
+"""
+IronicConf - file ``/etc/ironic/ironic.conf``
+=============================================
+This class provides parsing for the file ``/etc/ironic/ironic.conf``.
+
+Sample input data is in the format::
+    [DEFAULT]
+    auth_strategy=keystone
+    default_resource_class=baremetal
+    enabled_hardware_types=idrac,ilo,ipmi,redfish
+    enabled_bios_interfaces=no-bios
+    enabled_boot_interfaces=ilo-pxe,pxe
+    enabled_console_interfaces=ipmitool-socat,ilo,no-console
+    force_raw_images = true
+
+    [agent]
+    deploy_logs_collect=always
+    deploy_logs_storage_backend=local
+    deploy_logs_local_path=/var/log/ironic/deploy/
+
+    [cinder]
+    auth_url=http://1.1.1.1:5000
+    project_domain_name=Default
+    project_name=service
+    user_domain_name=Default
+
+Examples:
+    >>> conf = shared[IronicConf]
```

Comment:
You may have copied this from another parser, but insights-core no longer uses the `shared[IronicConf]` syntax to access the parser object.  Also you should add a test for your example docstrings which will indicate you may need to change your examples.  Here is an example test for your docstrings, that will be run during pytests:

```
def test_ironic_conf_docs():
    failed, total = doctest.testmod(
        ironic_conf,
        globs={
            'conf': IronicConf(context_wrap(ironic_content)),
        }
    )
    assert failed == 0
```